plugins/PowerDVD/__init__.py

```diff
@@ -20,58 +20,155 @@
+    ("GreenButton", "Green Button", "Green button on a remote control.", "{F10}"),
+    ("YellowButton", "Yellow Button", "Yellow button on a remote control.", "{F11}"),
+    ("BlueButton", "Blue Button", "Blue button on a remote control.", "{F12}"),
+    ("RedButton", "Red Button", "Red button on a remote control.", "{F9}"),
 ]
 
+ProgramManagementActions = [
+    ("Maximize", "Maximize", "Maximize the CyberLink PowerDVD program.", "{F5}"),
+    ("Minimize", "Minimize", "Minimize the CyberLink PowerDVD program.", "{Ctrl+n}"),
+    ("ToggleFullscreen", "Toggle Fullscreen", "Toggle playback to/from full screen mode.", "z"),
+    ("IncreaseScreenBrightness", "Increase Screen Brightness", "Increase screen brightness on supported displays.", "{Ctrl+Up}"),
+    ("DecreaseScreenBrightness", "Decrease Screen Brightness", "Decrease screen brightness on supported displays.", "{Ctrl+Down}"),
+    ("EjectDisc", "Eject Disc", "Eject the disc in the selected disc drive.", "{Ctrl+e}"),
+    ("Settings", "Settings", "Open the PowerDVD settings window.", "{Ctrl+Shift+c}"),
+    ("Help", "Help", "Open PowerDVD help.", "{F1}"),
+    ("About", "About", "Open the About PowerDVD window.", "{Ctrl+Shift+a}"),
+    ("AccessUpgradeInfo", "Access Upgrade Info", "Access PowerDVD upgrade information dialog.", "i"),
+    ("ClosePowerDVD", "Close PowerDVD", "Close PowerDVD.", "{Alt+F4}"),
+]
 
-gWindowMatcher = eg.WindowMatcher('PowerDVD{*}.exe', 'CyberLink PowerDVD{*}')
 
 
 class ActionPrototype(eg.ActionBase):
 
     def __call__(self):
-        hwnds = gWindowMatcher()
+        #
+        # Get the window handle (HWND) of the PowerDVD main window and the HWND of the PowerDVD player window.
+        #     When PowerDVD is running, the main window is not hidden and always has an HWND.
+        #     The player window is hidden and only exists when playback has begun.
+        # Combine the two lists, the main window being the first entry.
+        # If the list is empty, raise an error.
+        # Else, send the key stroke to the last element in the list
+        # (i.e., to the player window when playback has begun, else to the main window).
+        #
+        # This also works when PowerDVD is running in the background.
+        #
+        #
+        # AutoHotKey searches HWNDs differently, requiring only two lines of code to find the window
+        # and sending a key to it (also works when PowerDVD is in the background).
+        # AHK code for toggling play/pause:
+        #     ControlGet, OutputVar, Hwnd,,, PowerDVD
+        #     ControlSend, , {space}, ahk_id %OutputVar%
+        #
+
+	gWindowMatcherMainWindow = eg.WindowMatcher('PowerDVD{*}.exe', 'PowerDVD', None, None, None, 1, "false", 0, None)
+	gWindowMatcherPlayerWindow = eg.WindowMatcher('PowerDVD{*}.exe', 'PowerDVD', '{*}', '{*}', '{*}', 1, "true", 0, None)
+
+        hwnds = gWindowMatcherMainWindow() + gWindowMatcherPlayerWindow()
+
```

Comment:
`gWindowMatcherMainWindow` and `gWindowMatcherPlayerWindow` needs only to be instantiated once. So you can move them to module level (just like in v1.1)